template/.github/workflows/release.yml

```
@@ -0,0 +1,28 @@
+name: Netlify Release
```

Should we rename this file to `deploy.yml` to make it consistent with other projects?
```suggestion
name: Deploy to Netlify
```

---

1d366c0 ✅